packages/integration-tests/tests/retrospective-demo/step1-reflect.test.ts

```diff
@@ -0,0 +1,232 @@
+import config from '../config'
+import {test, expect} from '@playwright/test'
+import {goToNextPhase, startDemo} from './retrospective-demo-helpers'
+
+test.describe('retrospective-demo / reflect page', () => {
+  test('it shows an explanation popup', async ({page}) => {
+    await startDemo(page)
+
+    await expect(page.locator('[aria-label="Meeting tips"]')).toBeVisible()
+    await expect(
+      page.locator('[aria-label="Meeting tips"] :text("It Starts with Brutal Honesty")')
+    ).toBeVisible()
+    await expect(
+      page.locator(
+        '[aria-label="Meeting tips"] :text("When you’re ready to move on, hit the Next button below.")'
+      )
+    ).toBeVisible()
+  })
+
+  test('allows the user to enter feedback in start column', async ({page}) => {
+    await startDemo(page)
+
+    const startTextbox = '[data-cy=reflection-column-Start] [role=textbox]'
+    await page.click(startTextbox)
+    await page.type(startTextbox, 'Start doing this')
+    await page.press(startTextbox, 'Enter')
+
+    await expect(
+      page.locator('[data-cy="reflection-stack-Start"] :text("Start doing this")')
+    ).toBeVisible()
+  })
+
+  test('allows the user to enter feedback in the stop column', async ({page}) => {
+    await startDemo(page)
+
+    const stopTextbox = '[data-cy=reflection-column-Stop] [role=textbox]'
+    await page.click(stopTextbox)
+    await page.type(stopTextbox, 'Stop doing this')
+    await page.press(stopTextbox, 'Enter')
+
+    await expect(
+      page.locator('[data-cy="reflection-stack-Stop"] :text("Stop doing this")')
+    ).toBeVisible()
+  })
+
+  test('allows the user to enter feedback in the continue column', async ({page}) => {
+    await startDemo(page)
+
+    const continueTextbox = '[data-cy=reflection-column-Continue] [role=textbox]'
+    await page.click(continueTextbox)
+    await page.type(continueTextbox, 'Continue doing this')
+    await page.press(continueTextbox, 'Enter')
+
+    await expect(
+      page.locator('[data-cy="reflection-stack-Continue"] :text("Continue doing this")')
+    ).toBeVisible()
+  })
+
+  test('allows the user to delete previously entered feedback', async ({page}) => {
+    await startDemo(page)
+
+    const startTextbox = '[data-cy=reflection-column-Start] [role=textbox]'
+    await page.click(startTextbox)
+    await page.type(startTextbox, 'Start doing this')
+    await page.press(startTextbox, 'Enter')
+
+    await expect(
+      page.locator('[data-cy="reflection-stack-Start"] :text("Start doing this")')
+    ).toBeVisible()
+
+    await page.click(
+      '[data-cy="reflection-stack-Start"] [aria-label="Delete this reflection card"]'
+    )
+
+    await expect(
+      page.locator('[data-cy="reflection-stack-Start"] :text("Start doing this")')
+    ).not.toBeVisible()
+  })
+
+  test('displays simulated users writing reflections in the start column', async ({page}) => {
+    test.setTimeout(30_000)
```

Comment:
This test, and the ones below it, are quite slow because we check that the "writing reflections" text appears in all columns. I could rework this test for speed if this is too granular / not necessary to test.